pkg/controller/admin/events_test.go

```diff
@@ -162,88 +109,16 @@ func TestAdminEvents(t *testing.T) {
 	t.Run("lists_system", func(t *testing.T) {
 		t.Parallel()
 
-		mux := mux.NewRouter()
-		mux.Use(middlewares...)
-		mux.Handle("/", c.HandleEventsShow()).Methods(http.MethodGet)
-
 		ctx := ctx
 		ctx = controller.WithSession(ctx, &sessions.Session{})
 		ctx = controller.WithUser(ctx, &database.User{})
 
-		r := httptest.NewRequest(http.MethodGet, "/?realm_id=0", nil)
-		r = r.Clone(ctx)
-		r.Header.Set("Content-Type", "text/html")
-
-		w := httptest.NewRecorder()
-
-		mux.ServeHTTP(w, r)
-		w.Flush()
+		w, r := envstest.BuildFormRequest(ctx, t, http.MethodGet, "/", nil)
+		r = mux.SetURLVars(r, map[string]string{"realm_id": "0"})
+		handler.ServeHTTP(w, r)
 
 		if got, want := w.Code, http.StatusOK; got != want {
 			t.Errorf("Expected %d to be %d", got, want)
 		}
 	})
-
-	t.Run("renders", func(t *testing.T) {
-		t.Parallel()
-
-		_, _, session, err := harness.ProvisionAndLogin()
-		if err != nil {
-			t.Fatal(err)
-		}
-
-		cookie, err := harness.SessionCookie(session)
-		if err != nil {
-			t.Fatal(err)
-		}
-
-		eventTime, err := time.Parse(time.RFC3339, "2020-03-11T12:00:00Z")
-		if err != nil {
-			t.Fatal(err)
-		}
-		audit := &database.AuditEntry{
-			RealmID:       0, // system entry
-			Action:        "test action",
-			TargetID:      "testTargetID",
-			TargetDisplay: "test target",
-			ActorID:       "testActorID",
-			ActorDisplay:  "test actor",
-			CreatedAt:     eventTime,
-		}
-		harness.Database.RawDB().Save(audit)
-
-		// Create a browser runner.
-		browserCtx := browser.New(t)
-		taskCtx, done := context.WithTimeout(browserCtx, project.TestTimeout())
-		defer done()
-
-		if err := chromedp.Run(taskCtx,
-			// Pre-authenticate the user.
-			browser.SetCookie(cookie),
-
-			// Visit /admin
-			chromedp.Navigate(`http://`+harness.Server.Addr()+`/admin/events`),
-
-			// Wait for render.
-			chromedp.WaitVisible(`body#admin-events-index`, chromedp.ByQuery),
-
-			// Search from and hour before to and hour after our event
-			chromedp.SetValue(`#from`, eventTime.Add(-time.Hour).Format(rfc3339PartialLocal), chromedp.ByQuery),
```

Comment:
Same here. This exercised the search-by-time filter. Are we okay dropping these assertions?

Reply:
Fixed